.gitignore

```diff
@@ -74,3 +74,7 @@ node_modules
 _site/
 .sass-cache/
 .jekyll-metadata
+
+# secrets
+Resources/*.xcconfig
```

Comment:
Should we commit an example xcconfig to know the format for those unfamiliar?

Reply:
No worries! The script generates the files for you

Reply:
Ahh cool, sorry haven't run it yet!

Reply:
Should we be ignoring only `xcconfig`s in the Resources folder?